### PR TITLE
chore: december release notes and ABI update

### PR DESCRIPTION
**Pull request by devbww**
This change is [Reviewable](https://reviewable.io/reviews/googleapis/google-cloud-cpp/5566)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -42,6 +45,59 @@
...
+* feat(bigtable): create logging layer for Cloud Bigtable (#5515)
+* docs: document Bigtable thread safety (#5394)
+
+### Pub/Sub
+
+* feat(pubsub): by default Subscriber::Subscribe retries forever (#5552)
+* fix(pubsub): deadlock during streaming pull shutdown (#5547)
+* fix(pubsub): deadlock during fire & forget shutdown (#5541)
+* refactor!: PaginationRange<T> is now an alias to StreamRange<T> (#5538)
+* fix(pubsub): treat some kUnavailable as success (#5510)
```
Comment (coryan) [inline CHANGELOG.md:59]: That was removed as part of #5552, I would remove it, but up to you.

File: CHANGELOG.md [hunk 2]
```diff
@@ -42,6 +45,59 @@
...
+
+### Storage
+
+* refactor!: change gcs to use common PaginationRange<T> (#5545)
+* fix(storage): use channel options for credentials (#5524)
+* fix: validate service account credentials contain a usable key (#5404)
+
+### Common Libraries
+
+* fix: inspect returned grpc::Status (#5558)
```
Comment (coryan) [inline CHANGELOG.md:95]: I think that is a test-only fix, maybe remove?

**Review by coryan — APPROVED** [review]
A couple of minor nits, I think optional.

I just sent #5565 for review, would be nice to include in the release because it is affecting an internal user.

File: CHANGELOG.md (hunk 1 above)
Comment (devbww) [reply CHANGELOG.md]: Done.

File: CHANGELOG.md (hunk 2 above)
Comment (devbww) [reply CHANGELOG.md]: Done.

**Comment by devbww** [comment]
> I just sent #5565 for review, would be nice to include in the release because it is affecting an internal user.

Ack.

**Review by coryan — APPROVED** [review]

**Pull request merged**